### PR TITLE
Create section of latest Aptos releases

### PR DESCRIPTION
**Pull request by clay-aptos**

### CODE REVIEW

**Review by christian-aptos — APPROVED** [review]
It's a bit confusing to see the latest releases in the sidebar but not in the table.

**Comment by clay-aptos** [comment]
> It's a bit confusing to see the latest releases in the sidebar but not in the table.

Agreed. We need input from the team on where to get this information for v1.2.5 and all releases going forward. See my request for this in the related:
https://github.com/aptos-labs/aptos-core/issues/6704#issuecomment-1439004789

**Comment by thepomeranian** [comment]
Throwing an idea out there for how we can layout the table and side panel: https://excalidraw.com/#room=47432cee87433528a055,L6H6hipPJBrMl897iKE1fQ

**Review by davidiw — COMMENTED** [review]
Maybe let's add something to the main page too?

**Comment by clay-aptos** [comment]
> Maybe let's add something to the main page too?

You have got it. I will add a section on finding the latest Aptos releases toward the top.

**Comment by clay-aptos** [comment]
> Throwing an idea out there for how we can layout the table and side panel: https://excalidraw.com/#room=47432cee87433528a055,L6H6hipPJBrMl897iKE1fQ

I have implemented the subpages in the latest commit:
https://deploy-preview-6657--aptos-developer-docs.netlify.app/releases/aptos-releases

File: developer-docs-site/docs/releases/devnet-release.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+## Aptos CLI
+
+|Release | Docker Image Tag | Docker SHA-256 | Branch | Commit Hash|
+|---|---|---|---|---|
+|[Aptos CLI Release v1.0.7](https://github.com/aptos-labs/aptos-core/releases/tag/aptos-cli-v1.0.7)| `aptos-cli-v1.0.7` | `?` | ?| `?` |
+
+## Aptos Framework
+
+|Release | Docker Image Tag | Docker SHA-256 | Branch | Commit Hash|
```
Comment (perryjrandall) [inline developer-docs-site/docs/releases/devnet-release.md:20]: I don't think docker image tag is relevant here, instead maybe a link to the docs?

Also we should keep track here of which version is on which network

File: developer-docs-site/docs/releases/devnet-release.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+|---|---|---|---|---|
+|[Aptos Framework Release v1.2.0](https://github.com/aptos-labs/aptos-core/releases/tag/aptos-framework-v1.2.0)| `aptos-framework-v1.2.0` | `?` | ?| `?` |
+
+
+## Aptos Node
+
+|Release | Docker Image Tag | Docker SHA-256 | Branch | Commit Hash|
```
Comment (perryjrandall) [inline developer-docs-site/docs/releases/devnet-release.md:27]: Can we keep track here which is devnet testnet and mainnet?

**Review by perryjrandall — CHANGES_REQUESTED** [review]
Generally this is looking amazing!!!!

Just need a couple tweaks but this is sorely needed

File: developer-docs-site/docs/releases/devnet-release.md (hunk 2 above)
Comment (clay-aptos) [reply developer-docs-site/docs/releases/devnet-release.md:27]: > Can we keep track here which is devnet testnet and mainnet?

Thanks! Do the distinct pages for each network accomplish this? Or are you saying we need something about the other networks on each of those pages?
https://deploy-preview-6657--aptos-developer-docs.netlify.app/releases/mainnet-release
https://deploy-preview-6657--aptos-developer-docs.netlify.app/releases/testnet-release
https://deploy-preview-6657--aptos-developer-docs.netlify.app/releases/devnet-release

Thanks again!

File: developer-docs-site/docs/releases/devnet-release.md (hunk 1 above)
Comment (clay-aptos) [reply developer-docs-site/docs/releases/devnet-release.md]: I have removed the *Docker Image Tag* column from the Aptos Framework table on all three network release pages.

@perryjrandall , please help me understand what you mean by which network. Note we have a release page dedicated to each here.

**Comment by clay-aptos** [comment]
Hi all, I have now updated the PR to:

1. Address @perryjrandall 's feedback.
2. Update the Testnet Release page to include v.1.3.0 - https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.3.0
3. Update the Devnet Release page to reflect the new fields and values announced in Discord as a Node v2 table - 
https://discord.com/channels/945856774056083548/956692649430093904/1081050987554422924

Please help me populate, make accurate, or remove the remaining tables and fields. Thanks!

**Review by perryjrandall — APPROVED** [review]
LFG

**Comment by perryjrandall** [comment]
Looking great @clay-aptos !

**Pull request merged**